sdk/src/signature.rs

```diff
@@ -43,6 +43,15 @@ impl Keypair {
         self.0.to_bytes()
     }
 
+    pub fn from_base58_string(s: &str) -> Self {
+        Self::from_bytes(&bs58::decode(s).into_vec().unwrap()).unwrap()
+    }
+
+    pub fn to_base58_string(&self) -> String {
+        // Remove .iter() once we're rust 1.47+
```

Comment:
FYI: @t-nelson Another `LengthAtMost32` artificial shackle. I'm looking forward to #12739. :)

```
error[E0277]: arrays only have std trait implementations for lengths 0..=32
   --> sdk/src/signature.rs:51:22
    |
51  |         bs58::encode(&self.0.to_bytes()).into_string()
    |                      ^^^^^^^^^^^^^^^^^^ the trait `std::array::LengthAtMost32` is not implemented for `[u8; 64]`
    |
   ::: /home/.cargo/registry/src/github.com-1ecc6299db9ec823/bs58-0.3.1/src/lib.rs:207:18
    |
207 | pub fn encode<I: AsRef<[u8]>>(input: I) -> encode::EncodeBuilder<'static, I> {
    |                  ----------- required by this bound in `bs58::encode`
    |
    = note: required because of the requirements on the impl of `std::convert::AsRef<[u8]>` for `[u8; 64]`
    = note: required because of the requirements on the impl of `std::convert::AsRef<[u8]>` for `&[u8; 64]`

```